H/HandIsomorphism/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "HandIsomorphism"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kdub0/hand-isomorphism.git", "dabcee4a84c1d62ee6ded9b6ff02ece6823fcc0f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+install_license ${WORKSPACE}/srcdir/hand-isomorphism/LICENSE.txt
+cp hand-isomorphism/LICENSE.txt $prefix/share/licenses/LICENSE.txt
+cd hand-isomorphism/src/
+mkdir $includedir
+cp hand_index.h $includedir/hand_index.h
+OS=$(uname)
+if [[ "${target}" == *linux* ]] || [[ "${target}" == *freebsd* ]]; then     mkdir $prefix/lib/;     gcc -std=c99 -O2 -shared -o $prefix/lib/libhandisomorphism.so -fPIC deck.c hand_index.c -lm; elif [[ "${target}" == *mingw* ]]; then     mkdir $prefix/bin/;     gcc -std=c99 -O2 -shared -o $prefix/bin/libhandisomorphism.dll -fPIC deck.c hand_index.c -lm; elif [[ "${target}" == *apple* ]]; then          mkdir $prefix/lib/;     gcc -std=c99 -O2 -dynamiclib -o $prefix/lib/libhandisomorphism.dylib -fPIC deck.c hand_index.c -lm; fi
+exit
```

Comment:
This can be greatly simplified, we provide [environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables) to handle platform-specific differences seamlessly:
```suggestion
cd $WORKSPACE/srcdir/hand-isomorphism/src/
install_license ../LICENSE.txt
install -Dvm 644 hand_index.h ${includedir}/hand_index.h
mkdir -p "${libdir}"
cc -std=c99 -O2 -shared -o "${libdir}/libhandisomorphism.${dlext}" -fPIC deck.c hand_index.c -lm
```